helm-charts/opentaco/Chart.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v2
+name: opentaco
+description: OpenTaco - Complete Infrastructure-as-Code platform deployment
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
+
+# Umbrella chart that deploys all OpenTaco components
+# This chart orchestrates:
+#   - PostgreSQL database (optional - can use Cloud SQL instead)
+#   - Digger Backend (terraform orchestration backend)
+#   - Taco Statesman (IaC state management)
+#   - Drift Detection service
+#   - Taco UI (React frontend)
+
+dependencies:
+  # Optional PostgreSQL - disable if using Cloud SQL
+  - name: postgresql
+    version: "15.x.x"
+    repository: https://charts.bitnami.com/bitnami
+    condition: postgresql.enabled
+    tags:
+      - database
```

Comment:
dont think we should rely on bitnami here at all https://medium.com/@PlanB./the-death-of-free-bitnami-charts-and-what-it-means-for-your-infrastructure-8081344e2363